components/BuzzerPopup.jsx

```diff
@@ -0,0 +1,48 @@
+import { useTranslation } from 'react-i18next';
+import { useEffect, useState } from 'react';
+
+const BuzzerPopup = ({ buzzed }) => {
+  const { t } = useTranslation();
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    let timer;
+    if (buzzed && buzzed.id) {
+      var audio = new Audio('buzzer.mp3');
+      audio.play();
+      setIsVisible(true);
+      timer = setTimeout(() => {
+        setIsVisible(false);
+      }, 5000);
+    }
```

Comment:
I believe this would trigger anytime someone in the game buzzed. 
We I think we only want the sound on either

Just the first buzzer 
Just the first 2 buzzers

Imagine playing this will 15 people all buzzing at once. It would quickly get overwhelming.

Reply:
Audio plays and popup shows now only on first buzzer